src/System Application/App/Email/src/Email/Outbox/EmailEditor.Codeunit.al

```diff
@@ -162,18 +162,22 @@ codeunit 8906 "Email Editor"
     var
         FileName: Text;
         Instream: InStream;
-        AttachmentName, ContentType : Text[250];
-        AttachamentSize: Integer;
     begin
         UploadIntoStream('', '', '', FileName, Instream);
-        if FileName = '' then
-            exit;
+        if FileName <> '' then
+            UploadAttachmentFromStream(EmailMessageImpl, FileName, Instream);
+    end;
 
+    internal procedure UploadAttachmentFromStream(EmailMessageImpl: Codeunit "Email Message Impl."; FileName: Text; Instream: InStream)
```

Comment:
Just call it UploadAttachment, the FromStream comes from our legacy days when we could not do overrides, hence we had to come up with unique names. Now that we can do overrides, we should use them.